engine/engine/src/engine_main.cpp

```diff
@@ -15,7 +15,7 @@
 
 #include <dlib/time.h>
 #include <dlib/thread.h>
-#include <graphics/glfw/glfw.h>
+#include <dmsdk/graphics/glfw/glfw.h>
```

Comment:
For compile time reasons, I opted to add the glfw.h to be part of the sdk for now, as a "hidden" and undocumented member. Basically, what I wanted was some of its values in a wrapper header. This was the most convenient way forward.